make/git/git_pullall.sh

```diff
@@ -44,6 +44,8 @@ if test -z "${has_remote}"; then
 else
     echo "First, updating with rebase from remote ${branch}."
     git pull --rebase origin ${branch}
+    echo "\nSecond, updating submodules."
+    git submodule update --init --recursive
```

Comment:
Hmm, drmemory does this after the pull below; I think that is what we want, to reflect the remote main branch changes

Reply:
Done.